vmdb/app/helpers/application_helper/dialogs.rb

```diff
@@ -95,7 +95,7 @@ def radio_options(field, url, value)
 
     extra_options = {
       :onclick  => remote_function(
-        :with     => "miqSerializeForm(this)",
+        :with     => "miqSerializeForm($(this).parent().attr('id'))",
```

Comment:
longer term: Is it possible to search up for a class or something instead of saying "the forms parent" ?

Not sure if applicable, but that can often be very brittle.